index.html

```diff
@@ -182,42 +182,63 @@ <h2>About Me</h2>
     <div class="cards-grid-container">
       <ul>
         <li class="cards-description">
-            <img src="./assets/icon-languages.svg" alt="languages">
-            <h2>Languages</h2>
-            <ul>
-              <li>JavaScript</li>
-              <li>Ruby</li>
-              <li>HTML</li>
-              <li>CSS</li>
-            </ul>
+          <img src="./assets/icon-languages.svg" alt="languages">
+          <h2>Languages</h2>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
         </li>
 
         <li class="cards-description">
-            <img src="./assets/icon-frameworks.svg" alt="frameworks">
-            <h2>Frameworks</h2>
-            <ul>
-              <li>Bootstrap</li>
-              <li>Ruby on rails</li>
-              <li>RSpec</li>
-              <li>Capybara</li>
-              <li>Selenium</li>
-            </ul>
+          <img src="./assets/icon-frameworks.svg" alt="frameworks">
+          <h2>Frameworks</h2>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+            <li>Selenium</li>
+          </ul>
         </li>
 
         <li class="cards-description">
-            <img src="./assets/icon-skills.svg" alt="skills">
-            <h2>Skills</h2>
-            <ul>
-              <li>Codekit</li>
-              <li>GitHub</li>
-              <li>Codepen</li>
-              <li>Gitlab</li>
-              <li>Terminal</li>
-            </ul>
+          <img src="./assets/icon-skills.svg" alt="skills">
+          <h2>Skills</h2>
+          <ul>
+            <li>Codekit</li>
+            <li>GitHub</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
         </li>
       </ul>
     </div>
   </section>
+
+  <!-- Contact Form -->
+  <section class="contact-form">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form action="https://formspree.io/f/xwkyqzow" method="post">
+      <input type="text" name="name" placeholder="Full name" required>
+      <input type="email" name="email" placeholder="Email address" required>
+      <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here" required></textarea>
+      <button type="submit">Get in touch</button>
+    </form>
+    <div class="iconLinks">
+      <img src="./assets/twitter.svg" alt="twitter">
+      <img src="./assets/linkedin.svg" alt="linkedin">
+      <img src="./assets/medium.svg" alt="medium">
+      <img src="./assets/github.svg" alt="github">
+      <img src="./assets/angellist.svg" alt="angellist">
+    </div>
```

Comment:
- [ ] All this icon should be in `ul` & `li` tag,  & each `li` tag be inside `a` tag. Let's fix this